### PR TITLE
Closes #151 Council update

### LINKED ISSUE #151: Blog Post:  New pharmaverse council members announcement
### Blog Post

Won't spoil the surprise by naming them here, but @NNaikp will be preparing this and then i'll update the website after the news is shared.

### PR DESCRIPTION
**Pull request by NNaikp**
Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: `c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
- [x] Add a short description for your blog post in the `description` field at the top of the markdown document.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by NNaikp** [comment]
@rossfarrugia Would you like more details about the companies added?

**Comment by NNaikp** [comment]
@bms63 Seems like I don't have rights to assign reviewers and Development in this repo 🤷‍♂️

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-04-29_appsilon_and__sa..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two council seats on the pharmaverse council!
```
Comment (rossfarrugia) [inline posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:24]:
```suggestion
We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two open council seats on the pharmaverse council!
```

**Review by bms63 — APPROVED** [review]

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Appsilon will on the council be represented by **[Damian Rodziewicz](https://www.linkedin.com/in/drodziewicz/)** and Sanofi by **[Andre Couturier](https://www.linkedin.com/in/andre-couturier-1800b447/)**.
+
+# Appsilon and Open Source
+
+Appsilon is the maintainer of the [`rhino`](https://appsilon.github.io/rhino/) package already on the pharmaverse package rooster. Appsilon also powers the popular [ShinyConf](https://www.shinyconf.com/) and are involved in the pilot submissions to FDA through the R consortium working group ["R for submission"](https://rconsortium.github.io/submissions-wg/).
```
Comment (rossfarrugia) [inline posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:30]:
```suggestion
Appsilon is the maintainer of the [`rhino`](https://appsilon.github.io/rhino/) package already on the pharmaverse package roster, as well as having numerous employees contributing to pharmaverse in spaces such as TLG packages and our [Examples site](https://pharmaverse.github.io/examples/). Appsilon also powers the popular [ShinyConf](https://www.shinyconf.com/) and are involved in the pilot submissions to FDA through the R consortium working group ["R for submission"](https://rconsortium.github.io/submissions-wg/).
```

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+# Sanofi and Open Source
+
+Sanofi recently became a member of the R consortium and have in recent years invested heavily in the adoption of R internally. In line with the pharmaverse goal, Sanofi is committed to the adoption and further development of open source software in the pharma industry. This is evident through Sanofi's contributions to [`falcon`](https://pharmaverse.github.io/falcon/) and [`riskmetric`](https://pharmar.github.io/riskmetric/) and their [extension](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2024/Connect/US/Bethesda/PAP_ET10.pdf) of the [`teal`](https://insightsengineering.github.io/teal/latest-tag/) framework.
```
Comment (rossfarrugia) [inline posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:35]:
```suggestion
Sanofi recently became a member of the R consortium and have in recent years invested heavily in the adoption of R internally. In line with the pharmaverse goal, Sanofi is committed to the adoption and further development of open source software in the pharma industry. This is evident through Sanofi's contributions to [`falcon`](https://pharmaverse.github.io/falcon/) and [`riskmetric`](https://pharmar.github.io/riskmetric/) and their [extension](https://phuse.s3.eu-central-1.amazonaws.com/Archive/2024/Connect/US/Bethesda/PAP_ET10.pdf) of the [`teal`](https://insightsengineering.github.io/teal/latest-tag/) framework, which they are working towards open sourcing.
```

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+
```
Comment (rossfarrugia) [inline posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:37]: Maybe add a small section on why we accepted them, referencing the section on our site about expectations of council members, just to remind people of the high bar it takes to be a council member, and where to look for more details on what being on the council entails.

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+We are excited to announce that **Appsilon** and **Sanofi** will take up the remaining two council seats on the pharmaverse council!
+
+Appsilon will on the council be represented by **[Damian Rodziewicz](https://www.linkedin.com/in/drodziewicz/)** and Sanofi by **[Andre Couturier](https://www.linkedin.com/in/andre-couturier-1800b447/)**.
+
+# Appsilon and Open Source
```
Comment (rossfarrugia) [inline posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:28]: could be worth adding each company's logo just for some imagery to mix up the blog

**Review by rossfarrugia — COMMENTED** [review]
@NNaikp looks good to me, i left some comments/suggestions for conisderation. i dont think we need to add too much more on the companies themselves as anyone could google to learn more about them

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd (hunk 1 above)
Comment (NNaikp) [reply posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd]: Accepted this one in push

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd (hunk 2 above)
Comment (NNaikp) [reply posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd]: Accepted this one in push

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd (hunk 4 above)
Comment (NNaikp) [reply posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd:51]: Re-iterated what the companies are committing to with a link to our charter.

File: posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd (hunk 3 above)
Comment (NNaikp) [reply posts/2024-04-29_appsilon_and__sa.../appsilon_and__sanofi_joins_the_pharmaverse_council!.qmd]: Accepted this one in push

**Review by rossfarrugia — APPROVED** [review]

**Pull request merged**